go.mod

```diff
@@ -3,17 +3,18 @@ module github.com/charmbracelet/charm
 go 1.17
 
 require (
-	github.com/auth0/go-jwt-middleware/v2 v2.0.1
+	github.com/auth0/go-jwt-middleware/v2 v2.2.1
 	github.com/caarlos0/env/v6 v6.10.1
 	github.com/calmh/randomart v1.1.0
 	github.com/charmbracelet/bubbles v0.15.0
 	github.com/charmbracelet/bubbletea v0.23.2
-	github.com/charmbracelet/keygen v0.4.2
+	github.com/charmbracelet/keygen v0.5.0
 	github.com/charmbracelet/lipgloss v0.7.1
 	github.com/charmbracelet/log v0.2.2
 	github.com/charmbracelet/ssh v0.0.0-20221117183211-483d43d97103
 	github.com/charmbracelet/wish v1.1.1
 	github.com/dgraph-io/badger/v3 v3.2103.2
+	github.com/go-jose/go-jose v2.6.2+incompatible
```

Comment:
neat, i didn't know about +incompatible
resources
- https://utcc.utoronto.ca/~cks/space/blog/programming/GoModulePlusIncompatibleMeaning
- https://stackoverflow.com/questions/57355929/what-does-incompatible-in-go-mod-mean-will-it-cause-harm